dvc/command/experiments.py

```diff
@@ -800,45 +844,100 @@ class CmdExperimentsInit(CmdBase):
     DVCLIVE = "dvclive"
     DEFAULT_NAME = "default"
 
+    @post_processing(dict)
+    def init_interactive(self, defaults=None):
+        defaults = defaults or {}
+        prompts = {
+            "cmd": "[b]Command[/b] to execute",
+            "code": "Path to a [b]code[/b] file/directory",
+            "data": "Path to a [b]data[/b] file/directory",
+            "models": "Path to a [b]model[/b] file/directory",
+            "metrics": "Path to a [b]metrics[/b] file",
+            "params": "Path to a [b]parameters[/b] file",
+            "plots": "Path to a [b]plots[/b] file/directory",
+            "live": "Path to log [b]dvclive[/b] outputs",
+        }
+        message = (
+            "This command will guide you to set up your first stage in "
+            "[green]dvc.yaml[/green].\n"
+        )
```

Comment:
There were some suggestions to do `dvc init` if we detect that there's no `dvc` repo. `dvc exp init` is likely going to be used for the first time, so I added that up, but we could autodetect the presence of `dvc.yaml` file as well to have heading or not.

Reply:
Awesome, that sounds great!